spec/models/course_spec.rb

```diff
@@ -641,4 +641,80 @@
     subject { create(:course) }
     its(:recruitment_cycle_year) { should eq('2019') }
   end
+
+  describe '#enrichments' do
+    describe '#find_or_initialize_draft' do
+      let(:course) { create(:course, enrichments: enrichments) }
+
+      copyable_enrichment_attributes =
+        %w[
+          about_course
+          course_length
+          fee_details
+          fee_international
+          fee_uk_eu
+          financial_support
+          how_school_placements_work
+          interview_process
+          other_requirements
+          personal_qualities
+          qualifications
+          salary_details
+        ].freeze
+
+      subject { course.enrichments.find_or_initialize_draft }
+
+      context 'no enrichments' do
+        let(:enrichments) { [] }
+
+        it "sets all enrichment attributes to be nil" do
+          expect(subject.attributes.slice(*copyable_enrichment_attributes)).to eq copyable_enrichment_attributes.map { |key| [key, nil] }.to_h
```

Comment:
And then, with `enrichments_attributes` and a little more tweaking, this can be:

```suggestion
          expect(enrichment_attributes.values).to be_all(&:nil?)
```